app/processing/statement/components/get-detailed-funding.js

```diff
@@ -0,0 +1,17 @@
+const getFundings = require('../../funding')
+const getInvoiceLine = require('../../invoice-line')
+
+const getDetailedFunding = async (calculationId, paymentRequestId) => {
+  const fundings = await getFundings(calculationId)
+  const detailedFundings = []
+
+  for (const funding of fundings) {
+    const invoiceLine = await getInvoiceLine(funding.fundingCode, paymentRequestId)
+    const detailedFunding = { ...funding, ...invoiceLine }
+    detailedFundings.push(detailedFunding)
+  }
+
+  return detailedFundings

```

Comment:
Detailed funding also needs to include a way to identify what the funding is.

Example expected output of each funding element below:

```
"name":"Arable and horticultural soils",
         "level":"Introductory",
         "area":"5.00",
         "rate":"22.00",
         "annualValue":"110.00",
         "quarterlyValue":"27.50",
         "quarterlyReduction":"0.00",
         "quarterlyPayment":"27.50",
         "reductions":[]
```

Is missing the name, area and rate.
The reductions array will always be empty for now, but it's a mandatory field down stream